### ISSUE TITLE
labels and edges in BuildingNet v1 Data

### ISSUE THREAD

**Issue by mi-song** [buildingnet/buildingnet_dataset#15]
Are the labels provided in the point clouds of BuildingNet v1 inferred from a trained model rather than being ground truth data used for training? 
Also, how can one obtain the edge relationships between subgroups in the provided data?

**Comment by buildingnet** [comment]
Hi. The point cloud labels for the BuildingNet v1 are the ground truth annotations used for training. For the edge features between mesh subgroups you can use these `model_data/GNN` from the BuildingNet v0, since v1 is a subset of it. You will need to filter out the building models that are not part of v1 and substitute the subgroup group with the ones from v1. Moreover, for the MinkowskiNet features you should use the pretrained model based on the v1 dataset (you can find more instructions [here](https://github.com/buildingnet/buildingnet_dataset/tree/main/MinkowskiNet#export-features-for-buildinggnn)).
